resources/guest_configs/microvm-kernel-arm64-5.10-no-sve.config

```diff
@@ -2892,7 +2892,11 @@ CONFIG_HAVE_FTRACE_MCOUNT_RECORD=y
 CONFIG_HAVE_SYSCALL_TRACEPOINTS=y
 CONFIG_HAVE_C_RECORDMCOUNT=y
 CONFIG_TRACING_SUPPORT=y
-# CONFIG_FTRACE is not set
+CONFIG_FTRACE=y
+CONFIG_BOOTTIME_TRACING=y
+CONFIG_FUNCTION_TRACER=y
+CONFIG_FUNCTION_GRAPH_TRACER=y
+CONFIG_DYNAMIC_FTRACE=y
```

Comment:
do you need all the configs or just a few like CONFIG_FTRACE/CONFIG_FUNCTION_TRACER. It would be good to have more info on how you found these configs and if there is an external link which explains why/how these configs affect the bootup.

Reply:
I was trying different options and this is the smallest set that I could find.

Reply:
ok, do you think if it would be a good idea to create a TODO somewhere to find out how these options helped? Maybe someone who has time could try to figure this out?

Reply:
Not sure. 4.14 is an ancient kernel and investigating the underlying reason may be not reasonable in terms of effort to result ratio. What deserves a TODO is a task of removing this kernel once we drop 4.14 from supported list of kernels. But who knows when that will happen.